website/api/models/NewsletterSubscription.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * NewsletterSubscription.js
+ *
+ * @description :: A model definition represents a database table/collection.
+ * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
+ */
+
+module.exports = {
+
+  attributes: {
+
+    //  ╔═╗╦═╗╦╔╦╗╦╔╦╗╦╦  ╦╔═╗╔═╗
+    //  ╠═╝╠╦╝║║║║║ ║ ║╚╗╔╝║╣ ╚═╗
+    //  ╩  ╩╚═╩╩ ╩╩ ╩ ╩ ╚╝ ╚═╝╚═╝
+
+    emailAddress: {
+      type: 'string',
+      description: 'The email address that was provided when this subscription was created',
+      required: true,
+      unique: true,
+      isEmail: true,
+      maxLength: 200,
+      example: 'mary.sue@example.com'
+    },
+
+    isActive: {
+      type: 'boolean',
+      description: 'Whether this newsletter subscription is active or not.',
+      defaultsTo: true,
+    },
```

Comment:
1. Consider converting `isActive` to `isUnsubscribedFromAll` for clarity, which flips its meaning
2. I'd explicitly write out in the subscription what this means: "Whether this newsletter subscription has been updated to indicate a preference for unsubscribing from all current and future newsletters."